project/src/components/film-list/film-list.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+import { Film } from '../../types/film';
+import FilmCard from '../film-card/film-card';
+
+type FilmListProps = {
+  films: Film[];
+}
+
+function FilmList({films}: FilmListProps): JSX.Element {
+  const setActiveFilm = useState(0)[1];
```

Comment:
в целом можно и так, но лучше попробовать что-нибудь такое: 
```suggestion
  const [_, setActiveFilm] = useState(0);
```

неиспользуемую переменную обозначаем нижним подчеркиванием, распространенная практика, можно почитать в интернете.